src/app/design/frontend/base/default/template/payment/form/omisecc.phtml

```diff
@@ -117,7 +132,7 @@
             </select>
         </div>
 
-        <input data-omise="omise_token" type="hidden" name="payment[omise_token]">
+        <input id="<?php echo $_code ?>_token" data-omise="omise_token" type="hidden" name="payment[omise_token]">
```

Comment:
Now, the attribute, id, was used to indicate and manipulate so the attribute, data-omise, can be removed from all elements.